components/Common/Breadcrumbs/BreadcrumbRoot/index.tsx

```diff
@@ -0,0 +1,31 @@
+import HomeIcon from '@heroicons/react/24/outline/HomeIcon';
+import type { FC, PropsWithChildren } from 'react';
+
+import BreadcrumbItem from '@/components/Common/Breadcrumbs/BreadcrumbItem';
+import LocalizedLink from '@/components/LocalizedLink';
+
+import styles from './index.module.css';
+
+type BreadcrumbRootProps = {
+  hideHome?: boolean;
+};
+
+const BreadcrumbRoot: FC<PropsWithChildren<BreadcrumbRootProps>> = ({
+  hideHome = false,
+  children,
+}) => (
+  <nav aria-label="breadcrumb">
+    <ol className={styles.list}>
```

Comment:
I see that you haven't added the microdata from schema.org yet, are you going to eventually add it?